stock_demand_estimate/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Demand Estimate",
+    "summary": "Allows to create demand estimates.",
+    "version": "12.0.1.0.0",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
```

Comment:
Following https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py, you can add maintainers and development_status keys.

Reply:
But is optional

Reply:
installable attribute would be nice to have